layouts/shortcodes/added-in.html

```diff
@@ -1,8 +1,9 @@
-{{ $ver := .Params.v }}
-{{ $this := .Params.this }}
+{{- $ver := .Params.v -}}
 
-{{ if $this }}
-  <span><strong>[New in this version]</strong></span>
-{{ else }}
+{{- with page.Params.version -}}
```

Comment:
this block doesn't appear to be working:

![image](https://github.com/matrix-org/matrix-spec/assets/1190097/60776469-b4da-45ae-b846-7277bdb69179)